PHP/PHP Source.sublime-syntax

```diff
@@ -1307,14 +1307,14 @@ contexts:
           embed: scope:source.js
           embed_scope: meta.embedded.js source.js
           escape: (?=^\s*JAVASCRIPT\b)
-        - match: (<<<)\s*(JSON)\s*$\n?
+        - match: <<<\s*(JSON)\s*$\n?
```

Comment:
Maybe https://github.com/sublimehq/Packages/issues/2073

Scopeing everything punctuation seems sub optima. Already applied the RFC in several syntaxes.

Reply:
just a note for myself: check https://github.com/deathaxe/sublime-packages/commit/9927f07a2f70801b8cef13a2d5412e9a0f3b1a6f

Reply:
Changes are made in [`798c1fc` (#2822)](https://github.com/sublimehq/Packages/pull/2822/commits/798c1fc919f896739c050ec5b0dd85b000c62c9a)